sdk/openai/openai/src/api/requestOptionDeserializer.ts

```diff
@@ -0,0 +1,17 @@
+import { OperationOptions } from "@azure/core-client";
+import { RequestParameters } from "@azure-rest/core-client";
+
+export function deserializeOperationOptions(options: OperationOptions): RequestParameters {
```

Comment:
not sure if deserializeOperationOptions is the right operation name here?

Reply:
`operationOptionsToRequestParameters` maybe?